Project-I-Social-Card/social-card/src/index.css

```diff
@@ -1,4 +1,6 @@
 body {
+  width: 800px;
```

Comment:
Try not to set the height/width of the body directly. Try and set the dimensions for a container div instead.